provisioning/device/src/ProductInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Azure.Devices.Provisioning.Client
+{
+    // TODO: Unify ProductInfo with DeviceClient.
+    internal class ProductInfo
+    {
+        public string Extra { get; set; } = "";
+
+        public override string ToString()
+        {
+            const string Name = "Microsoft.Azure.Devices.Provisioning.Client";
+
+            // TODO:
+            // DO NOT EDIT the following line; it is updated by the bump_version script (https://github.com/Azure/iot-sdks-internals/blob/master/release/csharp/inputs.js)
+            const string Version = "1.5.2"; // CommonAssemblyVersion
```

Comment:
>const string Version = "1.5.2 [](start = 12, length = 29)

Why do even need this. Can't the code extract directly from the Assembly file?

Reply:
I was following the patterns from IoT Hub. That's a very good point. I'll add this to the TODO comment which was in place so we can update the bump_version script.